charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -159,6 +159,11 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--metricsport={{ .Values.provisioner.httpMetrics.containerPort }}"
             - "--metricspath=/metrics"
+{{- if .Values.provisioner.httpMetrics.mappingmetrics.enabled }}
+            - "--enablemappingmetrics={{ .Values.provisioner.httpMetrics.mappingmetrics.enabled }}"
+            - "--scrapeMetricsTimeout={{ .Values.provisioner.httpMetrics.mappingmetrics.timeout }}"
+            - "--scrapeMetricsGoroutines={{ .Values.provisioner.httpMetrics.mappingmetrics.goroutineNumber }}"
```

Comment:
I do not think a variable name like `*Goroutines` is very user-friendly. Users should need to care about how the parallelism is implemented. Maybe use `--scrapeMetricsJobs=` as option?

Reply:
done